backend/Node/routes/spotifyRoute.ts

```diff
@@ -159,4 +170,102 @@ router.get('/hello', async (req: any, res: any) => {
+	const trackIds = ['1ZEYuf4A7V3ikAQ8hQNS1O',
+	'2CVChktEKBsM6v4IfdFU5k',
+	'6bGMSP3H9YqkmaLnaJTIoF',
+	'7hZtfayYQukFySbj746HJL',
+	'4lKg94fHZZ3pozsNFTew2x',
+	'5ruzrDWcT0vuJIOMW7gMnW',
+	'5SAK4pJLz2Lu8RDerM5qiU',
+	'1K13OsIMc0HLgEJHZoWH78',
+	'5GQQ3qp1BgqCdI50n0GzNP',
+	'0h1YzxN6QTjEkJGU6fxsyt',
+	'2WyjObUjqZVqVxAX9PvO8t',
+	'73W5aXorr5vxrySFcoZqIN',
+	'7FqScuVJysPgwVFcepFJks',
+	'5IWlLl3xT95o8TSv3O8tRH',
+	'6wQXjA6KWbwPT3ydQCsJ4P',
+	'6v3wihX7hSonQaBVSYTBgW',
+	'7kpe2K0hdSssLj4G4YjxdU',
+	'6JqwuALkOqCYXMDO3llZUm',
+	'1AA3ZjLo9tD2iSZAs2svyj',
+	'4JkOsslKrWDYYb5dcft9AH'
+  ];
+  const popularity = [
+    39,
+    78,
+    84,
+    49,
+    54,
+    79,
+    51,
+    55,
+    35,
+    39,
+    5,
+    58,
+    48,
+    61,
+    59,
+    51,
+    61,
+    52,
+    56,
+    50
+  ];
+  // TODO: validate limit (Spotify's range: 1 to 100 inclusive)
+  const limit = 20;
+  /* END OF HARDCODED VALUES */
+
+  // TODO: move thiese endpoints somewhere nice, ML server port num should be set and loaded
+  const MLServerRes = await axios.post(`http://localhost:5000/recommend_playlist`, {
+	location,
+	pop,
+	clouds,
+	temp,
+	mood,
+	activity,
+	audio_features,
+	popularity
```

Comment:
popularity is already a field inside of audio_features (which is also what the ML server expects), so there's no need to send back popularity on it's own as well :)

Reply:
Yeah, I fixed it, no more popularity alone. All in audio_features array